assets/js/googlesitekit/widgets/datastore/constants.js

```diff
@@ -16,4 +16,7 @@
  * limitations under the License.
  */
 
+export const WIDGET_STYLES = [ 'boxes', 'composite' ];
+export const WIDGET_WIDTHS = [ 'quarter', 'half', 'full' ];
```

Comment:
I'd argue it's more useful here to create individual constants for them too? Using constants is always a good idea e.g. to prevent typos from staying undetected.

Maybe make these objects, so that code can use constants like `WIDGET_STYLES.BOXES` or `WIDGET_WIDTHS.QUARTER`? Alternatively individual constants all prefixed, as in `WIDGET_STYLE_BOXES` or `WIDGET_WIDTH_QUARTER`.

If we want to go the object approach and still use for validation, we could use `Object.values`.

Reply:
Oh, fair point. Let's go for `WIDGET_STYLES.BOXES`. I think it's a bit easier to understand the scoping then 👍